src/anonymousUserTracking/complexCriteria.test.ts

```diff
@@ -28,25 +31,32 @@ describe('complexCriteria', () => {
             ],
             total: 50,
             eventType: 'purchase'
-          },
-          {
-            dataFields: {
-              preferred_car_models: 'Honda',
-              country: 'Japan'
-            },
-            eventType: 'user'
           }
         ]);
       }
+      if (key === SHARED_PREFS_USER_UPDATE_OBJECT_KEY) {
+        return JSON.stringify({
+          dataFields: {
+            preferred_car_models: 'Honda',
+            country: 'Japan'
+          },
+          eventType: 'user'
```

Comment:
There's a lot of repetition of mock data (here as well as the Subaru data). Could we use constants where appropriate?

Reply:
I notice there's a good deal of repetition here... might be an opportunity to create a helper function or something to handle a lot of the repeated code.

Reply:
Also noticing repetition in the other test files, recommend helper functions to reduce DRY

Reply:
yeah good callout. i will get that fixed up 👍

Reply:
For posterity, handling this in a follow-up PR.